tsconfig.json

```diff
@@ -8,7 +8,7 @@
 	  "moduleResolution": "node",
 	  "outDir": "./out/server",
 	  "sourceMap": true,
-	  "target": "es6",
```

Comment:
No need to drop trailing commas from tsconfig — Typescript is smart enough to allow them, and they make diffs nicer.

Reply:
(I suppose you've excluded this file in the pre-commit config for exactly this reason)

Reply:
In fact that file is not excluded, another one with same name but different path is. This change was required by `check-json`.

That story around json-with-comments makes the json-check almost useless. Do you want me to exclude it and avoid the change? I guess that if it was broken we would have known anyway.

Reply:
done

Reply:
Yep, we'll see tsc failures if it stops understanding it.